pages/_main/_category/index.vue

```diff
@@ -20,14 +20,13 @@
 
 <script>
 import DHero from '@/components/base/DHero';
-import DirsToCardSections from '@/components/blocks/DirsToCardSections.vue';
-import FilesToSections from '@/components/blocks/FilesToSections.vue';
 
 export default {
   components: {
     DHero,
-    DirsToCardSections,
-    FilesToSections
+    DirsToCardSections: () =>
```

Comment:
Maybe add a comment explaining the lazy loading happening here. It may be useful for those not familiar with it